packages/lambdas/src/cw-patients-grouping.ts

```diff
@@ -0,0 +1,82 @@
+import { S3Utils } from "@metriport/core/external/aws/s3";
+import * as Sentry from "@sentry/serverless";
+import { SQSEvent } from "aws-lambda";
+import { chunk } from "lodash";
+import { capture } from "./shared/capture";
+import { getEnv, getEnvOrFail } from "./shared/env";
+
+// Keep this as early on the file as possible
+capture.init();
+
+// Automatically set by AWS
+const lambdaName = getEnv("AWS_LAMBDA_FUNCTION_NAME");
+const region = getEnvOrFail("AWS_REGION");
+// Set by us
+const inputQueueURL = getEnvOrFail("INPUT_QUEUE_URL");
+const outputQueueURL = getEnvOrFail("OUTPUT_QUEUE_URL");
+const orgListS3BucketName = getEnvOrFail("ORG_LIST_S3_BUCKET");
+const orgListS3Key = getEnvOrFail("ORG_LISTS_S3_KEY");
+
+const CQ_ORG_CHUNK_SIZE = 50;
+
+type SimpleOrg = {
+  Id: string;
+  Name: string;
+  States: string[];
+};
+
+let orgs: SimpleOrg[] = [];
+async function loadOrgList(): Promise<void> {
+  const s3Utils = new S3Utils(region);
+  const cqOrgsAsString = await s3Utils.getFileContentsAsString("test", "test");
```

Comment:
should this be `orgListS3BucketName, orgListS3Key`?

Reply:
ouch, good catch!

Reply:
Updated. This is a code that I built originally before we updated the plan, so this is actually not being used as of now... that's the scope of "v2" on [the ticket](https://github.com/metriport/metriport-internal/issues/1195)